src/core/mesh/qgsmeshlayer.h

```diff
@@ -182,18 +183,23 @@ class CORE_EXPORT QgsMeshLayer : public QgsMapLayer
     const QgsMesh *nativeMesh() const SIP_SKIP;
 
     /**
-     * Returns triangular mesh (NULLPTR before rendering)
-     *
+     * Returns triangular mesh (NULLPTR before rendering). If the parameter triangleSize is provided, among the base triangular mesh
```

Comment:
add a new line after first sentence. First sentence is \brief doxygen comment, rest is long description